extensions/gmsa-dc/v1/template-link.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "[concat(EXTENSION_TARGET_VM_NAME_PREFIX, copyIndex(EXTENSION_LOOP_OFFSET), 'gmsa-dc')]",
+	"type": "Microsoft.Resources/deployments",
+	"apiVersion": "[variables('apiVersionDeployments')]",
+	"dependsOn": [
+		"[concat('Microsoft.Compute/virtualMachines/', EXTENSION_TARGET_VM_NAME_PREFIX, copyIndex(EXTENSION_LOOP_OFFSET), '/extensions/cse', '-EXTENSION_TARGET_VM_TYPE-', copyIndex(EXTENSION_LOOP_OFFSET))]"
+	],
+	"copy": {
+		"count": "EXTENSION_LOOP_COUNT",
+		"name": "gmsa-dcExtensionLoop"
+	},
+	"properties": {
+		"mode": "Incremental",
+		"templateLink": {
+			"uri": "https://raw.githubusercontent.com/JeremyWx/windows-testing/master/extensions/gmsa-dc/v1/template.json",
```

Comment:
Same as above. I think the URL has been left behind from testing. Should be kubernetes-sigs/windows-testing/...